packages/kbn-openapi-generator/src/openapi_generator.ts

```diff
@@ -63,10 +63,11 @@ export const generate = async (config: GeneratorConfig) => {
       };
     })
   );
-  // If there are no operations or components to generate, skip this file
+
+  // If there are no operations or component schemas to generate, skip this file
```

Comment:
this fixes a bug where a file with parameter components would produce an empty .gen.ts file. This is because `generationContext.components` contained `parameters`.